packages/shorebird_cli/lib/src/shorebird_cli_command_runner.dart

```diff
@@ -8,6 +8,7 @@ import 'package:cli_completion/cli_completion.dart';
 import 'package:mason_logger/mason_logger.dart';
 import 'package:scoped_deps/scoped_deps.dart';
 import 'package:shorebird_cli/src/commands/commands.dart';
+import 'package:shorebird_cli/src/commands/patches/patches.dart';
```

Comment:
Minor but this shouldn't be needed if we update the `commands.dart` to export `patches.dart`.